packages/dashboard/pubspec.yaml

```
@@ -27,14 +28,5 @@ dev_dependencies:
-    path: ../stem
-  stem_cli:
-    path: ../stem_cli
-  stem_postgres:
-    path: ../stem_postgres
-  stem_redis:
-    path: ../stem_redis
-  stem_sqlite:
-    path: ../stem_sqlite
-  timezone: 0.11.0
+  artisanal: ^0.2.0
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)